### ISSUE TITLE
Issues running air gapped

### ISSUE THREAD

**Issue by donaldm** [eclipse/che#20742]
### Describe the bug

The use of image shas is causing problems when trying to run eclipse che airgapped. When I try to push the ubi8-minimal image using any of docker, podman, or skopeo i am getting a different sha256 sum and the image is not found in the airgap registry. This keeps the eclipse che workspace from starting.

I have tried downloading this image on a couple different machines, through a proxy and not, bundling it as an archive... and I can not find a mechanism to airgap the ubi8-minimal image and preserve the expected sha256 sum.

I have spent days on this issue without success.

Thank you

### Che version

7.38@latest

### Steps to reproduce

Try to airgap the ubi8-minimal image and preserve the sha256 digest. 

### Expected behavior

Should be able to air-gap required images.

### Runtime

other (please specify in additional context)

### Installation method

OperatorHub

### Environment

Linux

### Additional context

Deploying on RKE2 cluster.

**Comment by benoitf** [comment]
Hello,

Maybe you're not copying the multi-arch manifest but only the specific arch image


If I go on https://catalog.redhat.com/software/containers/ubi8-minimal/5c64772edd19c77a158ea216 and copy the Digest displayed in bottom right page like
`registry.access.redhat.com/ubi8-minimal@sha256:4d6547bbb7c5fe0f0d37b491c0c1975ff96bf4cf1c26dd665e3a5d70918b5564`

Then I start a local registry

```
$ docker run -p 5000:5000 registry:2
```

And I copy all digests of the multi-arch image (using --all)
```
skopeo copy --all --dest-tls-verify=false docker://registry.access.redhat.com/ubi8-minimal@sha256:4d6547bbb7c5fe0f0d37b491c0c1975ff96bf4cf1c26dd665e3a5d70918b5564 docker://localhost:5000/florent:test
Getting image source signatures
Copying blob be961ec68663 done
Copying blob d46336f50433 done
Copying config e7685639f5 done
Writing manifest to image destination
Storing signatures
```

If I try to pull the digest from the local registry it's working
```
$ docker pull localhost:5000/florent:test@sha256:4d6547bbb7c5fe0f0d37b491c0c1975ff96bf4cf1c26dd665e3a5d70918b5564      localhost:5000/florent@sha256:4d6547bbb7c5fe0f0d37b491c0c1975ff96bf4cf1c26dd665e3a5d70918b5564: Pulling from florent
Digest: sha256:4d6547bbb7c5fe0f0d37b491c0c1975ff96bf4cf1c26dd665e3a5d70918b5564
Status: Image is up to date for localhost:5000/florent@sha256:4d6547bbb7c5fe0f0d37b491c0c1975ff96bf4cf1c26dd665e3a5d70918b5564
localhost:5000/florent:test@sha256:4d6547bbb7c5fe0f0d37b491c0c1975ff96bf4cf1c26dd665e3a5d70918b5564
```


So probably you're comparing the 'multi-arch manifest' digest and the 'specific arch digest' or you were not copying the multi-arch manifest but the specific arch image

**Comment by donaldm** [comment]
Thank you for the quick response. I think that makes sense. I had tried the copy all, but it is possible I was trying the wrong digest. 

I ended up solving my issue by relying on my mirror configuration to Artifactory to pull the images from quay.io and then setting variables for the plugin and devfile images. At some point I may need to run this fully air-gapped so this information will be useful then.

Thank you again!
